### ISSUE TITLE
Windows Cheat sheet 

### ISSUE THREAD

**Issue by shadyhamsters** [OlivierLaflamme/Cheatsheet-God#1]
Any chance thered be a release for a windows big vulnerabilities of the sort equivalent to the solaris cheatsheet.

all hail king olivier!

**Comment by OlivierLaflamme** [comment]
yes I have a few things currently in the works. Sorry for having been a ghost. Im back.

**Comment by OlivierLaflamme** [comment]
yeah for sure!:grin:

**Comment by OlivierLaflamme** [comment]
@MrTsRex yeah for sure! :grin:
